### PR TITLE
[20439] Authentication Handshake Properties documentation

### PR DESCRIPTION
**Pull request by Mario-DL**
This PR adds the documentation for configuring the `Authentication Handshake` behavior through the following properties:
* `max_handshake_requests`
* `initial_handshake_resend_period`
* `handshake_resend_period_gain` 

And also, fills the `PropertyPolicy Security` page by remembering the configurable properties of the different security plugins

We may backport this to 2.13.x 2.12.x 2.10.x and 2.6.x

### CODE REVIEW

**Comment by Mario-DL** [comment]
@richiprosima please test docs

File: docs/fastdds/property_policies/security.rst [hunk 1]
```diff
@@ -1,10 +1,132 @@
...
+|DomainParticipantQos::properties-api|
+``dds.sec.crypto.plugin`` with the value ``builtin.AES-GCM-GMAC``.
+Moreover, this plugin needs the activation of the :ref:`auth-pki-dh`.
+The DDS\:Crypto\:\AES-GCM-GMAC plugin is configured using the :ref:`access-permissions`, i.e the cryptography
+plugin is configured through the properties and configuration files of the access control plugin.
+For further information and examples in this regard please refer to :ref:`crypto-aes-gcm-gmac`.
+
+Logging plugin settings
+***********************
+The DDS\:Logging\:DDS_LogTopic authentication plugin, can be activated setting the |DomainParticipantQos|
```
Comment (elianalf) [inline docs/fastdds/property_policies/security.rst:128]:
```suggestion
The :ref:`DDS\:Logging\:DDS_LogTopic <logging-logtopic>` authentication plugin, can be activated setting the |DomainParticipantQos|
```

**Review by elianalf — CHANGES_REQUESTED** [review]
It would be nice if there was a reference to the new page `Security Plugin Settings` at the end of the section [8. Security](https://eprosima-fast-rtps--681.org.readthedocs.build/en/681/fastdds/security/security.html) so it can be easly found

**Comment by Mario-DL** [comment]
> Everything is clear! It would be nice if there was a reference to the new page `Security Plugin Settings` at the end of the section [8. Security](https://eprosima-fast-rtps--681.org.readthedocs.build/en/681/fastdds/security/security.html) so it can be easly found

Good one !

**Comment by Mario-DL** [comment]
@richiprosima please test docs

**Comment by Mario-DL** [comment]
@richiprosima please test docs

**Comment by Mario-DL** [comment]
@richiprosima please test docs

**Comment by Mario-DL** [comment]
@richiprosima please test docs

**Review by elianalf — APPROVED** [review]

**Comment by JesusPoderoso** [comment]
@Mergifyio backport 2.13.x 2.12.x 2.10.x 2.6.x

**Pull request merged**